docs/README.md

```diff
@@ -2,6 +2,16 @@
 
 This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
 
+
+## Prerequisites
+```console
+npm install
+npm run docs:build
+```
+
+Before you start a local development, Make sure the required command in the root package to generate an api file. If not, You may encounter "Not Found Error" regarding an api file.
```

Comment:
How about using more simple explanation like this:
```
Before starting a local development, run `docs:build` in the root directory to generate API documents.
```

Reply:
Oh, I didn't know that, That is more simple than mine, I'll do re - Pull requests with same simple explanations in  the [CONTRIBUTING.md](https://github.com/naver/egjs-flicking/blob/master/CONTRIBUTING.md) file, thank you for your reply.